src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+enum class Prize(
```

Comment:
Using enum 👍 
Perhaps the next thing to think about is how to handle 2nd place logic.